Shared/UI/Views/MessageContentView/MessageContentView.swift

```diff
@@ -26,11 +26,13 @@ class MessageContentView: BaseView {
     }
 
     // Sizing
+    static let bubbleHeight: CGFloat = 168
```

Comment:
This is the largest bubble height that works on the mini devices. Any larger and the text input will run into the message at 3-4 lines.